flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/serialization/KinesisDeserializationSchema.java

```diff
@@ -37,13 +37,15 @@
 	 *
 	 * @param recordKey the records's key as a byte array (null if no key has been set for the record)
 	 * @param recordValue the record's value as a byte array
-	 * @param stream the name of the Kinesis stream that this record was sent to
+	 * @param partitionKey the record's partition key at the time of writing.
 	 * @param seqNum the sequence number of this record in the Kinesis shard
 	 * @param approxArrivalTimestamp the server-side timestamp of when Kinesis received and stored the record
+	 * @param stream the name of the Kinesis stream that this record was sent to
+	 * @param shardId The identifier of the shard the record was sent to.
```

Comment:
nit: some param descriptions end with period, some don't. I think the others all don't end with period, let's keep consistent with that.